GitCommands/CommitTemplateItem.cs

```diff
@@ -56,8 +56,7 @@ public static void SaveToSettings(CommitTemplateItem[] items)
         public static CommitTemplateItem[] LoadFromSettings()
         {
             string serializedString = AppSettings.CommitTemplates;
-            bool shouldBeUpdated;
-            var templates = DeserializeCommitTemplates(serializedString, out shouldBeUpdated);
+            var templates = DeserializeCommitTemplates(serializedString, out var shouldBeUpdated);
```

Comment:
:memo: Refactoring to use `out T` should be sent as a single pull request with no other changes